packages/developer-console-spa/src/components/ConfigureSearch/SearchWizardForm.tsx

```diff
@@ -0,0 +1,186 @@
+import {
+  Button,
+  Card,
+  CardBody,
+  Flex,
+  FlexItem,
+  Stack,
+  StackItem,
+  Text,
+} from '@patternfly/react-core';
+import { createStore, useStateMachine } from 'little-state-machine';
+import { useCallback, useContext, useEffect, useMemo, useState } from 'react';
+import { updateSearchConfigService } from 'services/searchConfig';
+import { AppContext } from '../../context/AppContext';
+import useSearchConfig from '../../hooks/useSearchConfig';
+import Header from '../Header';
+import Loader from '../Loader';
+import ConfigureSearchStep1 from './Steps/Step1';
+import ConfigureSearchStep2 from './Steps/Step2';
+import ConfigureSearchStep3 from './Steps/Step3';
+import saveState, { overrideState } from './Steps/store';
+import { clamp, isEmpty } from 'lodash';
+import { CSSProperties } from 'react';
+
+createStore(
+  {
+    appId: '',
+    savedStepId: 0,
+    formData: {},
+  },
+  {
+    name: 'SearchConfiguration',
+  }
+);
+
+const steps = [
+  { label: 'API Endpoint', component: ConfigureSearchStep1 },
+  { label: 'Auth & Headers', component: ConfigureSearchStep2 },
+  { label: 'Field Mapping', component: ConfigureSearchStep3 },
+];
+
+export default function SearchWizardForm(props: any) {
+  const { appId, loading: appLoading } = useContext(AppContext);
+  const { searchConfig, loading: searchConfigLoading } = useSearchConfig(appId);
+  const [currentStepId, setCurrentStepId] = useState<number>(0);
+  const { actions, state } = useStateMachine({ saveState, overrideState });
+
+  const [defaultValues, setDefaultValues] = useState<App.SearchConfig>();
+
+  useEffect( () => {
+    setCurrentStepId( state.savedStepId ?? 0 );
+  }, [state.savedStepId] );
+
+  useEffect(() => {
+    if (!!searchConfig) {
```

Comment:
Can we change this to `Boolean(searchConfig)`

Reply:
Updated.